Sources/VitaminSwiftUI/Components/Button/VitaminButton.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  Vitamin iOS
+//  Apache License 2.0
+//
+
+#if arch(x86_64) || arch(arm64)
+import SwiftUI
+import VitaminCore
+
+@available(iOS 13, *)
+public struct VitaminButton: View {
+    var text: String = ""
```

Comment:
I did the change, and it implies several other changes in code.

Once done, I ask myself if it has really sense, since I do not see the point of passing nil as text.
If it is just to display an icon, the icon alone button is provided, in which text is unused, whatever its value.

I let you check my changes, maybe there is a better way to handle that.